resources/views/matchdays/edit.blade.php

```diff
@@ -84,8 +84,8 @@
             </form>
         </div>
         <div class="card-footer row justify-content-around">
-            <a href="/alletermine" class="col-5 btn btn-sm footer-buttons" type="button">Speichern</a>
-            <a href="/alletermine" class="col-5 btn btn-sm footer-buttons text-danger" type="button">Löschen</a>
+            <a href="{{ url('/alletermine') }}" class="col-5 btn btn-sm footer-buttons" type="button">Speichern</a>
```

Comment:
I wont comment on these again but try to search for all instances of `url() ` in your templates an replace it with `route()`